packages/ui/src/components/widgets/WidgetContainer.tsx

```diff
@@ -69,65 +69,50 @@ const WidgetContainer: FC<WidgetContainerProps> = ({
-        {children({
-          addressModalOpen,
-          setAddressModalOpen
-        })}
-        {isMounted ? (
-          <SwapModal
-            open={transactionModalOpen}
-            onOpenChange={(open) => {
-              onSwapModalOpenChange(open)
-              setTransactionModalOpen(open)
-            }}
-            fromToken={fromToken}
-            toToken={toToken}
-            amountInputValue={amountInputValue}
-            amountOutputValue={amountOutputValue}
-            debouncedInputAmountValue={debouncedInputAmountValue}
-            debouncedOutputAmountValue={debouncedOutputAmountValue}
-            tradeType={tradeType}
-            useExternalLiquidity={useExternalLiquidity}
-            address={address}
-            recipient={recipient}
-            isCanonical={useExternalLiquidity}
-            timeEstimate={timeEstimate}
-            onAnalyticEvent={onAnalyticEvent}
-            onSuccess={onSwapSuccess}
-            invalidateBalanceQueries={invalidateBalanceQueries}
-          />
-        ) : null}
-        <CustomAddressModal
-          open={addressModalOpen}
-          toAddress={customToAddress ?? address}
-          isSolanaSwap={isSolanaSwap}
-          onAnalyticEvent={onAnalyticEvent}
+      {children({
+        addressModalOpen,
+        setAddressModalOpen
+      })}
+      {isMounted ? (
+        <SwapModal
+          open={transactionModalOpen}
           onOpenChange={(open) => {
-            setAddressModalOpen(open)
-          }}
-          onConfirmed={(address) => {
-            setCustomToAddress(address)
-          }}
-          onClear={() => {
-            setCustomToAddress(undefined)
+            onSwapModalOpenChange(open)
+            setTransactionModalOpen(open)
           }}
+          fromToken={fromToken}
+          toToken={toToken}
+          amountInputValue={amountInputValue}
+          amountOutputValue={amountOutputValue}
+          debouncedInputAmountValue={debouncedInputAmountValue}
+          debouncedOutputAmountValue={debouncedOutputAmountValue}
+          tradeType={tradeType}
+          useExternalLiquidity={useExternalLiquidity}
+          address={address}
+          recipient={recipient}
+          isCanonical={useExternalLiquidity}
+          timeEstimate={timeEstimate}
+          onAnalyticEvent={onAnalyticEvent}
+          onSuccess={onSwapSuccess}
+          invalidateBalanceQueries={invalidateBalanceQueries}
         />
-        <WidgetFooter />
-      </Flex>
+      ) : null}
+      <CustomAddressModal
+        open={addressModalOpen}
+        toAddress={customToAddress ?? address}
+        isSolanaSwap={isSolanaSwap}
+        onAnalyticEvent={onAnalyticEvent}
+        onOpenChange={(open) => {
+          setAddressModalOpen(open)
+        }}
+        onConfirmed={(address) => {
+          setCustomToAddress(address)
+        }}
+        onClear={() => {
+          setCustomToAddress(undefined)
+        }}
+      />
+      {/* <WidgetFooter /> */}
```

Comment:
Did we loose the widget footer? We should probably remove this code

Reply:
Yep we did. Will remove the comment 👍